src/test/java/org/fsg1/fmms/backend/resources/MainTest.java

```diff
@@ -0,0 +1,10 @@
+package org.fsg1.fmms.backend.resources;
+
+import org.junit.Test;
+
+public class MainTest {
+    @Test
+    public void testStartingServer() {
+        Main.startServer();
```

Comment:
This will actually start the server on port 8080. 
Please do not do this!! This will collide with running services.

Reply:
It might be useful for integration tests later to run the whole app once. Another port is useful though yeah.